torchvision/prototype/transforms/_augment.py

```diff
@@ -99,8 +96,11 @@ def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
             return input
 
     def forward(self, *inputs: Any) -> Any:
-        if torch.rand(1) >= self.p:
-            return inputs if len(inputs) > 1 else inputs[0]
+        sample = inputs if len(inputs) > 1 else inputs[0]
+        if has_any(sample, features.BoundingBox, features.SegmentationMask):
+            raise TypeError(f"BoundingBox'es and SegmentationMask's are not supported by {type(self).__name__}()")
+        elif torch.rand(1) >= self.p:
```

Comment:
Nit: Sometimes you do if/elif, other times below if/if. Choose one idiom for consistency.